Sources/SQLKit/Query/SQLAlterTable.swift

```diff
@@ -31,7 +32,11 @@ public struct SQLAlterTable: SQLExpression {
                 $0.append(column)
             }
             for column in self.modifyColumns{
-                $0.append("MODIFY")
+                if dialect.name == "mysql" {
+                    $0.append("MODIFY")
+                } else {
+                    $0.append("ALTER COLUMN")
+                }
```

Comment:
Shouldn't we have a `SQLDialect.alterColumn` property, instead of just checking the dialect name?